recipes/default.rb

```diff
@@ -24,18 +24,18 @@
 
 Chef::Resource.send(:include, Chocolatey::Helpers)
 
-install_ps1 = File.join(Chef::Config['file_cache_path'], 'install.ps1')
+install_ps1 = File.join(Chef::Config['file_cache_path'], 'chocolatey-install.ps1')
 
-template install_ps1 do
+remote_file install_ps1 do
   action :create
   backup false
-  source 'InstallChocolatey.ps1.erb'
-  variables :download_url => node['chocolatey']['install_vars']['chocolateyDownloadUrl']
+  source 'https://chocolatey.org/install.ps1'
+  not_if { chocolatey_installed? && (node['chocolatey']['upgrade'] == false) }
```

Comment:
I agree there are people who want to run without a network.

Is getting them to copy the chocolatey.org/install.ps1 to a local system too much trouble?  They already have to have copies of the packages they want someplace local.

I do agree that this should bump the cookbook's major version, though, as semver.org recommends.  It is a breaking change for people without network.